src/test/compile-fail/E0253.rs

```diff
@@ -14,6 +14,8 @@ mod foo {
     }
 }
 
-use foo::MyTrait::do_something; //~ ERROR E0253
+use foo::MyTrait::do_something;
+    //~^ ERROR E0253
+    //~|NOTE not directly importable
```

Comment:
Updated it now, with a new commit, but in hindsight I wonder if I should have done a commit --amend?, and it might cause a merge conflict?

(apologies, I have got so used to working via gerrit, and github stumps me).